src/main/java/jenkins/plugins/office365connector/Office365ConnectorWebhookNotifier.java

```diff
@@ -38,7 +38,7 @@
 public final class Office365ConnectorWebhookNotifier {
 
     private static final Gson gson = new GsonBuilder().setFieldNamingPolicy(FieldNamingPolicy.IDENTITY)
-            .setPrettyPrinting().create();
+            .disableHtmlEscaping().setPrettyPrinting().create();
```

Comment:
did you check what happens now when the message has html opening or closing tag?

Reply:
and for sure that must be note why escaping is disabled

Reply:
If a message had html tags before this change it would have been escaped anyway, this allows html in messages.

Reply:
you are expecting to disable escaping for message only while this filed is used for whole model